be/src/Project/Services/Catalog/Catalog.Api/Services/Categories/CategoriesService.cs

```diff
@@ -379,6 +379,20 @@ public async Task<CategorySchemasServiceModel> GetCategorySchemasAsync(GetCatego
             };
         }
 
+        public async Task<bool> CategoriesSchemasImplementAttributeAsync(CategoriesSchemasImplementAttributeServiceModel model)
+        {
+            var query = $@"
```

Comment:
**suggestion:** can't we use the almost exact query that I sent via email? In my view it won't work for example when one deletes the 'length' key because it doesn't have any definitions, it's just a number